index.html

```diff
@@ -0,0 +1,193 @@
+        </div>
+        <div class="stories1">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image2">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image3">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories1">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+</section>
+<section class="contact-form">
+    <h2>
+        Let's chat!
+    </h2>
+    <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it. 
+    </p>
+    <form  action="https://formspree.io/f/mnqwoyza" method="POST">
+        <input id="name" name="Name" class="text" type="text" placeholder="Full Name" maxlength="30" required>
+        <input id="email" name="email" class="text" type="email"  placeholder="example@gmail.com" required>
+        <textarea name="Message" id="message" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+        <button type="button" id="btn" class="button">See Project</button>
+    </form>
+</section>
+<div class="icon">
+    <img src="images/Vector5.png" alt="line" class="img" id="icon">
```

Comment:
- [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.